QUARKUS-5623.md

```diff
@@ -0,0 +1,36 @@
+# QUARKUS-5623 - Logging in OpenTelemetry
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-5623
+
+Upstream PRs:
+    - https://github.com/quarkusio/quarkus/pull/38239
```

Comment:
Fix the formatting, see for example https://github.com/quarkus-qe/quarkus-test-plans/blob/0ecb4d376d284867df7db238b75df4e5b62c2dc2/QUARKUS-5623.md

Reply:
You've linked this very file as an example :-), but I got it and fixed it.